apps/mymedicare_cb/models.py

```diff
@@ -51,170 +47,254 @@ def get_and_update_user(subject, mbi_hash, hicn_hash, first_name, last_name, ema
+                log_dict["status"] = "FAIL"
+                log_dict["mesg"] = err_msg
+                logger.info(log_dict)
                 raise BBMyMedicareCallbackCrosswalkUpdateException(err_msg)
         else:
             # Previously stored value was None/Null and mbi_hash != None, update just the mbi hash.
             if mbi_hash is not None:
                 err_msg = "UPDATE mbi_hash since previous value was NULL"
-                log_get_and_update_user(auth_flow_dict, "OK", user, fhir_id, mbi_hash, hicn_hash, hash_lookup_type, err_msg)
+                log_dict["status"] = "OK"
+                log_dict["mesg"] = err_msg
+                logger.info(log_dict)
                 user.crosswalk.user_mbi_hash = mbi_hash
                 user.crosswalk.user_id_type = hash_lookup_type
                 user.crosswalk.save()
 
         # Update hash type used for lookup, if it has changed from last match.
         if user.crosswalk.user_id_type != hash_lookup_type:
-            err_msg = "UPDATE user_id_type as it has changed from the previous lookup value"
-            log_get_and_update_user(auth_flow_dict, "OK", user, fhir_id, mbi_hash, hicn_hash, hash_lookup_type, err_msg)
+            err_msg = (
+                "UPDATE user_id_type as it has changed from the previous lookup value"
+            )
+            log_dict["status"] = "OK"
+            log_dict["mesg"] = err_msg
+            logger.info(log_dict)
             user.crosswalk.user_id_type = hash_lookup_type
             user.crosswalk.save()
 
         err_msg = "RETURN existing beneficiary record"
-        log_get_and_update_user(auth_flow_dict, "OK", user, fhir_id, mbi_hash, hicn_hash, hash_lookup_type, err_msg)
+        log_dict["status"] = "OK"
+        log_dict["mesg"] = err_msg
+        logger.info(log_dict)
         return user, "R"
     except User.DoesNotExist:
         pass
 
-    user = create_beneficiary_record(username=subject,
-                                     user_hicn_hash=hicn_hash,
-                                     user_mbi_hash=mbi_hash,
-                                     fhir_id=fhir_id,
-                                     first_name=first_name,
-                                     last_name=last_name,
-                                     email=email,
-                                     user_id_type=hash_lookup_type,
-                                     auth_flow_dict=auth_flow_dict)
+    user = create_beneficiary_record(
+        username=subject,
+        user_hicn_hash=hicn_hash,
+        user_mbi_hash=mbi_hash,
+        fhir_id=fhir_id,
+        first_name=first_name,
+        last_name=last_name,
+        email=email,
+        user_id_type=hash_lookup_type,
+        request=request,
+    )
 
     err_msg = "CREATE beneficiary record"
-    log_get_and_update_user(auth_flow_dict, "OK", user, fhir_id, mbi_hash, hicn_hash, hash_lookup_type, err_msg)
+    log_dict["status"] = "OK"
+    log_dict["mesg"] = err_msg
```

Comment:
```suggestion
    log_dict["status"] = "OK"
    log_dict["mesg"] = err_msg
    log_dict["crosswalk"] = {
        "id": user.crosswalk.id,
        "user_hicn_hash": user.crosswalk.user_hicn_hash,
        "user_mbi_hash": user.crosswalk.user_mbi_hash,
        "fhir_id": user.crosswalk.fhir_id,
        "user_id_type": user.crosswalk.user_id_type,
    },

```

I think the crosswalk should be added here for compatibility.  Was experimenting with the code above, but getting an odd JSON output in the logs.

It has brackets where the previous logging and the same code in an above section does not.  Here's an example:

```
  "mesg": "CREATE beneficiary record",
  "crosswalk": [
    {
      "id": 10,
      "user_hicn_hash": "5aae85715a1d5aec0ded28505b7da3f58b4ebe83b76370477adf69b444038fe9",
      "user_mbi_hash": "279deecc2f3e08ea3445ac9efa9f816083fa23b23257b3ff8bfebb4f9d6a180e",
      "fhir_id": "-19990000000003",
      "user_id_type": "M"
    }
  ]
}
```
This is how it looks using the master branch:

```
  "hash_lookup_type": "M",
  "crosswalk": {
    "id": 2,
    "user_hicn_hash": "3637b48c050b8d7a3aa29cd012a535c0ab0e52fe18ddcf1863266b217adc242f",
    "user_mbi_hash": "37c37d08d239f7f1da60e949674c8e4b5bb2106077cb0671d3dfcbf510ec3248",
    "fhir_id": "-19990000000001",
    "user_id_type": "M"
  },

```
```

Reply:
agree that should prevent logging less...